backend/package.json

```diff
@@ -37,6 +37,7 @@
     "nanoid": "^3.1.20",
     "nodemailer": "^6.5.0",
     "response-time": "^2.3.2",
+    "supervisor": "^0.12.0",
```

Comment:
Where are you using this?

Reply:
As I mentioned earlier under the heading of other information in the pull request template, I had installed the package so that server automatically restarts once  I make some changes to the code. But I will remove it right away.

Reply:
We already have nodemon for that.
Use `npm run dev`